contracts/DrawCalculator.sol

```diff
@@ -27,7 +27,7 @@ contract DrawCalculator is IDrawCalculator {
     ITicket public immutable ticket;
 
     /// @notice The stored history of draw settings.  Stored as ring buffer.
-    IPrizeDistributionSplitter public immutable prizeDistributionSplitter;
+    IPrizeDistributionSource public immutable prizeDistributionSource;
```

Comment:
This shouldn't be public.  There's already a getter.

Reply:
Done: https://github.com/pooltogether/v4-core/pull/256/commits/7799c597adfb01be2de66bbed4427d15e9ed68c5